src/lib.rs

```diff
@@ -198,6 +200,27 @@ fn build_poi(row: Row) -> Option<Poi> {
         return None;
     }
 
+    let properties = build_poi_properties(&row, &name).unwrap_or(vec![]);
+
+    let weight_wikidata = properties
+        .iter()
+        .find(|p| &p.key == "wikidata")
+        .map_or(0., |_p| 100.0);
+
+    let names_count = properties
+        .iter()
+        .filter(|p| Regex::new(r"name:*").unwrap().is_match(&p.key))
```

Comment:
why not using [starts_with](https://doc.rust-lang.org/std/string/struct.String.html#method.starts_with) instead of adding a dependency ?